ci/docker-registry

```diff
@@ -1 +1 @@
-projects.registry.vmware.com
+docker.io
```

Comment:
nit: could you add back the newline char at the end of the file?

Reply:
Updated, PTAL.